extensions/blocks/map/edit.js

```diff
@@ -193,23 +193,31 @@ class MapEdit extends Component {
 							/>
 						</PanelBody>
 					) : null }
-					{ 'wpcom' !== apiKeySource && (
-						<PanelBody title={ __( 'Mapbox Access Token', 'jetpack' ) } initialOpen={ false }>
-							<TextControl
-								label={ __( 'Mapbox Access Token', 'jetpack' ) }
-								value={ apiKeyControl }
-								onChange={ value => this.setState( { apiKeyControl: value } ) }
-							/>
-							<ButtonGroup>
-								<Button type="button" onClick={ this.updateAPIKey } isDefault>
-									{ __( 'Update Token', 'jetpack' ) }
-								</Button>
-								<Button type="button" onClick={ this.removeAPIKey } isDefault>
-									{ __( 'Remove Token', 'jetpack' ) }
-								</Button>
-							</ButtonGroup>
-						</PanelBody>
-					) }
+					<PanelBody title={ __( 'Mapbox Access Token', 'jetpack' ) } initialOpen={ false }>
```

Comment:
I was a bit surprised to see the panel there, but empty:

![image](https://user-images.githubusercontent.com/426388/74358610-0a4c7b00-4dc2-11ea-87f2-b2c796c2c698.png)

Is there a specific reason why we don't remove it altogether?

Edit: nevermind, I saw the discussion in the comments now. :)